mods/gen2/items.js

```diff
@@ -30,6 +30,8 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",
+		shortDesc: "If holder is a Chansey, it removes all crit boosting effects, and crit ratio = 3",
```

Comment:
This is not the desc/shortDesc I told you to use...

Also, the long desc should probably be:

`If held by a Chansey, its critical hit ratio is set to +3 stages (for a crit chance of 33%), overriding all other crit rate changes.`